src/app/shared/generics/model/field.model.ts

```diff
@@ -1,9 +1,11 @@
 import {Crud} from './crud.model';
+import { FieldOption } from './field-option';
 
 export interface Field {
   code: string;
   type: string;
   label: string;
   required?: boolean;
   crud?: Crud;
+  options?: FieldOption[];
```

Comment:
Can you briefly explain the intent with this field? How is it meant to be used in general?

Reply:
When creating fields that have multiple options (radio/checkbox), values and labels for given options can be configured separately. This way, user sees a meaningful label (eg. FTN) and on submit value that server understands is sent (eg. Regular).